test/test_pebble.py

```diff
@@ -2028,6 +2029,18 @@ def test_wait_change_error(self):
             ('GET', '/v1/changes/70/wait', {'timeout': '4.000s'}, None),
         ])
 
+    def test_wait_change_socket_timeout(self):
+        def timeout_response(n: float):
+            self.time.sleep(n)
+            raise socket.timeout(504, "socket.timeout: timed out")
```

Comment:
This wouldn't be a 504 at this point, so how about:
```suggestion
            raise socket.timeout("socket.timeout: timed out")
```